Models/Quiz.cs

```diff
@@ -21,5 +23,30 @@ public class Quiz
         public Folder? Folder { get; set; } = null;
         public string? UserId { get; set; } = null;
 
+        public int TimesSolved { get; set; } = 0;
+
+        public DateTime? LastPlayed { get; set; } = null;
+
+        public int? Ao3Score { get; set; } = null;
```

Comment:
Do we need to save Average Of 3 score to the database? If it's derived from past scores we can make it similar how you did with `public List<int> PastScores` and save the hassle of calculating it in other places like https://github.com/xDziugas/Acuedify/pull/16/files#diff-9fa5fb003bc6d204dbee4820ecadeed5a1341ecb3a096190f4ecaa187eb28ccdR139-R147